### PR TITLE
Clarify /rules/lib/builtins/repository_ctx description of default working_directory for execute()

### LINKED ISSUE #21897: [bazel.build] Problem with /rules/lib/builtins/repository_ctx description of default working_directory for execute()
### Page link:

https://bazel.build/rules/lib/builtins/repository_ctx

### Problem description (include actual vs expected text, if applicable):

The description of working_directory for repository_ctx.execute() says:

    default is ""
    Working directory for command execution. Can be relative to the repository root or absolute.

but this does not actually state what the default of "" _means_.  I'm _guessing_ that the default of "" means that the default working directory for command execution is the root of the repository since it does not begin with a "/" that would mark it as an absolute path.

### Where do you see this issue? (include link to specific section of the page, if applicable)

https://bazel.build/rules/lib/builtins/repository_ctx#parameters_3

### Any other information you'd like to share?

_No response_

### PR DESCRIPTION
**Pull request by dws**
Fixes #21897 per feedback from @meteorcloudy.

### CODE REVIEW

File: src/main/java/com/google/devtools/build/lib/bazel/repository/starlark/StarlarkBaseExternalContext.java [hunk 1]
```diff
@@ -1561,7 +1561,8 @@ private static String profileArgsDesc(String method, List<String> args) {
             named = true,
             doc =
                 "Working directory for command execution.\n"
-                    + "Can be relative to the repository root or absolute."),
+                    + "Can be relative to the repository root or absolute.\n"
+                    + "The default is relative to the repository root."),
```
Comment (meteorcloudy) [inline src/main/java/com/google/devtools/build/lib/bazel/repository/starlark/StarlarkBaseExternalContext.java:1565]: Maybe just "The default is the repository root."?

**Review by meteorcloudy — CHANGES_REQUESTED** [review]
Thanks!

File: src/main/java/com/google/devtools/build/lib/bazel/repository/starlark/StarlarkBaseExternalContext.java (hunk 1 above)
Comment (dws) [reply src/main/java/com/google/devtools/build/lib/bazel/repository/starlark/StarlarkBaseExternalContext.java]: Done!

**Review by meteorcloudy — APPROVED** [review]
Thank you!

**Pull request closed without merging**